### PR TITLE
PMM-8250 Add UI test for GC PostgreSQL

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

**Comment by IvanaBrecherova** [comment]
Passed on jenkins pipeline: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/9206/tests

**Comment by IvanaBrecherova** [comment]
Test passed: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/9214/tests

**Review by denisok — APPROVED** [review]

File: tests/remoteInstances/remoteInstancesHelper.js [hunk 1]
```diff
@@ -152,6 +157,13 @@ module.exports = {
         password: process.env.AZURE_POSTGRES_PASS,
       },
     },
+    gc: {
+      gc_postgresql: {
+        address: process.env.GCP_SERVER_IP,
+        userName: process.env.GCP_USER,
+        password: process.env.GCP_USER_PASSWORD,
```
Comment (puneet0191) [inline tests/remoteInstances/remoteInstancesHelper.js:164]: try using secret here, I have tested it for my changes and it works fine.

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/remoteInstances/remoteInstancesHelper.js (hunk 1 above)
Comment (denisok) [reply tests/remoteInstances/remoteInstancesHelper.js:164]: @puneet0191 should it be general approach for all vars in this helper ? some wrapper ofr class, or whatever works in this framework ?

Comment (IvanaBrecherova) [reply tests/remoteInstances/remoteInstancesHelper.js:164]: just like this?
`        address: secret(process.env.GCP_SERVER_IP),
        userName: secret(process.env.GCP_USER),
        password: secret(process.env.GCP_USER_PASSWORD),`

Comment (puneet0191) [reply tests/remoteInstances/remoteInstancesHelper.js]: @denisok yes, will think about a generic approach, not needed for all vars, some variables we use only to check the job which triggered etc

**Review by denisok — APPROVED** [review]

**Review by puneet0191 — APPROVED** [review]

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**